---
permalink: /projects/ 
title: "Projects"
header:
  overlay_color: "#5e616c"
  overlay_image: /assets/images/plates.jpg
  overlay_filter: 0.15
toc: true
toc_sticky: true
author_profile: false

---

# UC San Diego

## Engineering Industrial Strains for Diacid Production

## Genome-Scale Modeling

>Broddrick JT, **Welkie DG**, Jallet D, Golden SS, Peers G, Palsson BO. Predicting the metabolic capabilities of *Synechococcus elongatus* PCC 7942 adapted to different light regimes. Metabolic Engineering. 2018 Nov 13;52:42-56.<br/> 
Links to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30439494){:target="_blank"} , [DOI](https://doi.org/10.1016/j.ymben.2018.11.001){:target="_blank"}, or<a class="btn btn-success btn-lg" href="https://drive.google.com/open?id=1jJub7N5iiOAv4gPGpZ0kkDLXqO2AWwvn" target="_blank" ><i class="far fa-file-pdf"></i>open PDF

>Broddrick JT¹ , Rubin BE¹, **Welkie DG**¹ , Du D, Mih N, Diamond S, Lee JJ, Golden SS, Palsson BO. Unique attributes of cyanobacterial metabolism revealed by improved genome-scale metabolic modeling and essential gene analysis. Proceedings of the National Academy of Sciences. 2016. Dec 113;51:E8344-E8353. ¹co-1st authorship <br/>  Links to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/27911809){:target="_blank"}, [DOI](https://doi.org/10.1073/pnas.1613446113){:target="_blank"}, or<a class="btn btn-success btn-lg" href="https://drive.google.com/open?id=1g8nSXTIxjOnFs6LV8sU_zyxZlm0jNWZj" target="_blank" ><i class="far fa-file-pdf"></i>open PDF

>**Welkie DG**, Golden SS, Palsson BO. "Creating a predictive metabolic model of photoautotrophic metabolism in cyanobacteria to improve bioengineering design." Poster Presentation at the UC San Diego Chancellor’s Research Excellence Annual Symposium. University of California, San Diego, CA. October 25, 2018 <a class="btn btn-success btn-lg" href="https://drive.google.com/open?id=1Tjz-dU8d6TEGqH61Dzez6PnUo0GYBqW-" target="_blank" > <i class="far fa-file-pdf"></i> view poster.
  
## KaiA and Circadian Clock Regulation
This project has been huge and rewarding. If my graduate work taught me anything it was that understanding the fitness of KO strains was critical and we as a community were basically in the dark about what genes were important for growth of phototrophic bacteria in light-dark cycles. The combination of massive mutant library screens with targeted molecular biology experiemtns proved a fruitful was to study the relationship between the circadian clock in the model cyanobacterium Synechococcus elongatus sp. PCC 7942 and the regulation of metabolic processes, especially those important for diurnal growth.
To identify the genetic contributors to fitness under light-dark cycles and acquire a more comprehensive picture of the regulatory network, we used an unbiased population-based screen that tracks the fitness contributions of individual S. elongatus genes to reproduction in diel cycles. The method, random bar-code transposon-site sequencing (RB-TnSeq), quantitatively compares the changes in abundances of individuals in a pooled mutant population over time. This assay allowed the calculation of a fitness contribution to LD cycle survival for each nonessential gene in the genome by leveraging multiple inactivating mutations spread across a genomic locus. 
>**Welkie DG**, Rubin BE, Chang Y, Smith S, LiWang A, Golden S. Genome-wide fitness assessment during diurnal growth reveals an expanded role of the cyanobacterial circadian clock protein KaiA. Proceedings of the National Academy of Sciences. 2018 July  115;30:E7174-E7183.<br/> Links to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29991601){:target="_blank"} 

These efforts along with those of some great colleagues led to the production of a review article that served as a summary of recent knowledge dealing with cyanobacteria and their lifestyle in light-dark cycling conditions.
>**Welkie DG**, Rubin BE, Diamond S, Hood RD, Savage DF, Golden SS. A hard day’s night: Cyanobacteria in diel cycles. Trends in Microbiology. 2018 December In Press *Corrected Proof* <br/>
Links to article: [DOI](https://doi.org/10.1016/j.tim.2018.11.002){:target="_blank"}
  
## Genetic Fitness and Interaction Screens

>Rubin BE, Huynh TN, **Welkie DG**, Diamond S, Simkovsky R, Pierce EC, et al. High-throughput interaction screens illuminate the role of c-di-AMP in cyanobacterial nighttime survival. PLoS Genetics. 2018 Apr 14;4:e1007301 <br/>
Links to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/29608558){:target="_blank"}, [DOI](https://doi.org/10.1371/journal.pgen.1007301){:target="_blank"}, or:<a class="btn btn-success btn-lg" href="https://drive.google.com/open?id=1YF5qtAZzlKsyA7ic3LZSWBq-Pe6_eSSJ" target="_blank" ><i class="far fa-file-pdf"></i>open PDF
  
# Purdue

## Modifying Carbohydrate Storage and Strain Physiology

My graduate career contributions focused on genetics and physiology of cyanobacteria for industrial applications. My dissertation titles “An integrated approach to develop unicellular cyanobacteria for biofuel production” described how to improve the physiological characteristics of the nitrogen fixing cyanobacterium Cyanothece sp. PC 7822, where I enhanced growth and culture population homogeneity was through nutrient adjustment. 

>**Welkie DG**, Sherman DM, Chrisler WB, Orr G, Sherman LA. Analysis of carbohydrate storage granules in the diazotrophic cyanobacterium *Cyanothece* sp PCC 7822. Photosynthesis Research 2013 Nov 118(1-2):25-36. <br/> 
Links to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/24142038){:target="_blank"}, [DOI](https://doi.org/10.1007/s11120-013-9941-z){:target="_blank"}, or<a class="btn btn-success btn-lg" href="https://drive.google.com/open?id=1JCrYEhuD9F8kURterbRSE07OF3k3k4--" target="_blank" ><i class="far fa-file-pdf"></i>open PDF
  
>**Welkie DG**, Lee BH, Sherman LA. Altering the structure of carbohydrate storage granules in the cyanobacterium *Synechocystis* sp. strain PCC 6803 through branching-enzyme truncations. Journal of Bacteriology 2015 Dec 198:701-710. <br/> 
Links to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/26668264){:target="_blank"}, [DOI](https://doi.org/10.1128/JB.00830-15){:target="_blank"}, or<a class="btn btn-success btn-lg" href="https://drive.google.com/open?id=1QE9B8kivtV29aVOk0OXDP0jNN2_bJV_Q" target="_blank" ><i class="far fa-file-pdf"></i>open PDF


## Photosynthetic Isoprene Biosynthesis

Lastly, participated in a project to engineer Synechocystis to produce the C5 compound, isoprene, and validated computational model predictions to optimize productivity.

>**Welkie DG**, Saha R, Zhang X, Maranas CD, and Sherman LA. “Optimizing isoprene production in Synechocystis 6803 utilizing computational modeling and transcriptional regulation”. Poster 11th Workshop on Cyanobacteria. Washington University, St. Louis, MO. 2013 <a class="btn btn-success btn-lg" href="https://drive.google.com/open?id=1-zaVjOPzuPZxNSSGlnsWKzLhuLacNZ-0" target="_blank" > <i class="far fa-file-pdf"></i> view poster.
  
## Gene Expression in Light-Dark Cycles

I also performed transcriptomics and proteomics across the diurnal cycle that detailed the synchronicity of transcriptional and translational processes.

>**Welkie D**, Zhang X, Markillie ML, Taylor R, Orr G, Jacobs J, Bhide K, Thimmapuram J, Gritsenko M, Mitchell H, Smith RD, Sherman LA. Transcriptomic and proteomic dynamics in the metabolism of a diazotrophic cyanobacterium, *Cyanothece* sp. PCC 7822 during a diurnal light-dark cycle. BMC Genomics 2014 Dec 15:1185.8 <br/> 
Links to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/25547186){:target="_blank"},[DOI](https://doi.org/10.1186/1471-2164-15-1185){:target="_blank"}, or<a class="btn btn-success btn-lg" href="https://drive.google.com/open?id=1wFPTqduSg5XnYQjx3QbliGQbiuaZBXMS" target="_blank" ><i class="far fa-file-pdf"></i>open PDF


# UW-Madison

## Microbial Community Dynamics to Environmental Disturbance
My first research project was under the supervision of Dr. Paul Weimer at the USDA Forage Research Center at the University of Wisconsin Madison. Understanding microbial community dynamics in the rumin of dairy cows is vital to understanding animal health, productivity, and has food safety consequences.
In this study we used automated ribosomal intergenic spacer analysis to track changes in the microbial community during the feeding cycle of dairy cows. Using two animals eating the same diet we found large changes in the associated liquid as well as in ruminal pH and VFA concentrations while we observed that solid-associated microbial communities displayed less change in composition within and across feeding cycles. To sample the ruminal microbiome, the cows were fistulated and, using a arm-length long glove, I used a small cup to retrieve a rumin sample over multile timepoints. Fun stuff!

>**Welkie DG**, Stevenson DM, Weimer PJ. ARISA analysis of ruminal bacterial community dynamics in lactating dairy cows during the feeding cycle. Anaerobe 2010 Apr 16;2:94-100.<br/> 
Links to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/19615457){:target="_blank"}

Following this I worked on a Northern Temperate Lakes Microbial Observatory Project to study the microbial communities in fresh water lakes working under the supervision of Dr. Katherine McMahon and Dr. Ashley Shade. 
This project set out to separate the environmental drivers of microbial communities in freshwater ecosystems in response to seasonal and abrupt disturbances. My role in both these projects was to collect the samples from the field, perform DNA extractions and assist with data analysis and aide in manuscript writing. <br/> 

>Shade A, Read JS, **Welkie DG**, Kratz TK, Wu CH, McMahon KD. Resistance, resilience and recovery: Aquatic bacterial dynamics after water column disturbance. Environmental Microbiology 2011 Oct 13;10:2752-2767. <br/> 
Links to article: [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/21883795){:target="_blank"}


 xcerpt: 'An integrated approach to develop unicellular cyanobacteria for biofuel production” described how to improve the physiological characteristics of the nitrogen fixing cyanobacterium Cyanothece sp. PC 7822, where I enhanced growth and culture population homogeneity was through nutrient adjustment, I also performed transcriptomics and proteomics across the diurnal cycle that detailed the synchronicity of transcriptional and translational processes, and I created mutant strains of Synechocystis sp. PCC 6803 that had modified carbohydrate metabolism to study the genetic rationale for the variation of stored carbohydrate granule morphologies across species. Lastly, participated in a project to engineer Synechocystis to produce the C5 compound, isoprene, and validated computational model predictions to optimize productivity.'


I am a molecular biologist who can't get enough of working with and learning about microbial metabolism and genetics. I earned my BS in **microbiology** at the University of Wisconsin-Madison and was fortunate enough to participate in some research projects studying microbioal ecology while there which ultimately instill in me a passion for science and the potienital of microbial biotechnology. 

e focused on analysis of microbial communities in different environments using 16S rRNA. I began my research experience under the supervision of Dr. Paul Weimer at the USDA Forage Research Center at the University of Wisconsin Madison were I used my knowledge of molecular biology and microbiology to study microbial community dynamics in dairy cows across feeding cycles. The result of this project was a first authored paper that was the first to report how solid-associated microbial communities show less change in composition within and across feeding cycles than liquid-associated bacteria using automated ribosomal intergenic spacer analysis. I next focused on microbial communities in fresh water lakes working under the supervision of Dr. Katherine McMahon and Dr. Ashley Shade on a Northern Temperate Lakes Microbial Observatory Project. This project set out to separate the environmental drivers of microbial communities in freshwater ecosystems in response to seasonal and abrupt disturbances. This project, too, resulted in a co-authored manuscript. My role in both these projects was to collect the samples from the field, perform DNA extractions and assist with data analysis and manuscript writing. 

My graduate career contributions focused on genetics and physiology of cyanobacteria for industrial applications. My dissertation titles “An integrated approach to develop unicellular cyanobacteria for biofuel production” described how to improve the physiological characteristics of the nitrogen fixing cyanobacterium Cyanothece sp. PC 7822, where I enhanced growth and culture population homogeneity was through nutrient adjustment, I also performed transcriptomics and proteomics across the diurnal cycle that detailed the synchronicity of transcriptional and translational processes, and I created mutant strains of Synechocystis sp. PCC 6803 that had modified carbohydrate metabolism to study the genetic rationale for the variation of stored carbohydrate granule morphologies across species. Lastly, participated in a project to engineer Synechocystis to produce the C5 compound, isoprene, and validated computational model predictions to optimize productivity.

For my postdoctoral career contributions, I want to focus on studying the relationship between the circadian clock in the model cyanobacterium Synechococcus elongatus sp. PCC 7942 and the regulation of metabolic processes, especially those important for diurnal growth.
I Experienced working with a variety of microbial strains, many of which are photosynthetic, to develop ways for engineering bacterial metabolism for biotechnology. I enjoy working on teams to leverage multiple skill sets on common goals and sharing my research with others. In addition to thriving on the bench, I have much experience mentoring students and fellow scientists on experimental design, techniques, and project management. A common drive in my work experience has been to take advantage of microbial metabolism for bioproduction — whether that be in producing consistent milk from dairy cows by studying rumen microbes in Wisconsin as an undergraduate, producing hydrogen and isprenoids in different cyanobacterial strains as a graduate student at Purdue, or leveraging the powerful regulatory influence of the circadian clock on gene expression in the model cyanobacterium Synechococcus to produce hearty strains for industrial cultivation and improved fitness as a postdoc at UCSD working with Susan Golden and Bernhard Palsson. 

This passion has led me to develop skills in DNA assembly and genetic engineering for targeted investigations, utilizing different -omics approaches and Tn-library screening for a systems level discovery, and computational tools to model the system and predict metabolic flux and essential reactions for different goals. 

